tests/examples/opinions/united_states/md_example.compare.json

```diff
@@ -235,7 +235,7 @@
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
     "docket_numbers": "47/14",
-    "judges": "Per Curiam",
```

Comment:
I'm not sure we want to not capture `per curiam` actually 

I think we actually want to capture that - and I assume PC Order means per curiam

Reply:
> I assume PC Order means per curiam
I think so too

Populating the "judges" field with "Per Curiam" seems improper. We actually have a Opinion.per_curiam field in the DB